openid-4-verifiable-presentations-1_0.md

```diff
@@ -232,7 +232,14 @@ Presentation of Verifiable Credentials using OpenID for Verifiable Presentations
 
 The Authorization Request follows the definition given in [@!RFC6749] taking into account the recommendations given in [@!I-D.ietf-oauth-security-topics].
 
-The Verifier may send an Authorization Request as Request Object by value or by reference as defined in JWT-Secured Authorization Request (JAR) [@RFC9101].
+The Verifier MAY send an Authorization Request as a Request Object either by value or by reference, as defined in the JWT-Secured Authorization Request (JAR) [@RFC9101].
+
+The Wallet may want to provide to the Verifier details about its technical capabilities to
+allow the Verifier to generate a request that matches the technical capabilities of that Wallet.
+To enable this, the Authorization Request can contain a `request_uri_method` parameter with the value `post`
+that signals to the Wallet that it can make an HTTP POST request to the Verifier's `request_uri`
+endpoint with information about its capabilities. The Wallet MAY continue with JAR
+when it receives `request_uri_method` parameter with the value `post` but does not support this feature.
```

Comment:
```suggestion
This specification defines a new mechanism for the cases when the Wallet wants to provide to the Verifier details about its technical capabilities to
allow the Verifier to generate a request that matches the technical capabilities of that Wallet.
To enable this, the Authorization Request can contain a `request_uri_method` parameter with the value `post`
that signals to the Wallet that it can make an HTTP POST request to the Verifier's `request_uri`
endpoint with information about its capabilities as defined in {#request_uri_method_post}. The Wallet MAY continue with JAR
when it receives `request_uri_method` parameter with the value `post` but does not support this feature.
```

Can we please make it clear that this paragraph describes a new mechanism defined by this specification?

Also, i think this deserves a bullet point in  https://openid.github.io/OpenID4VP/openid-4-verifiable-presentations-wg-draft.html#section-4

Reply:
also here is probably a good place to establish terminology. the first request from the verifier with `request_uri_mode` is authorization request. what is the POST request from the wallet? wallet capabilities request? probably better to reuse the same terminology as JAR: "the wallet retrieves Request Object using POST request"..